src/metabase/driver/duckdb.clj

```diff
@@ -28,6 +26,34 @@
 
 (driver/register! :duckdb, :parent :sql-jdbc)
 
+(def premium-features-namespace
+  (try
+    (require '[metabase.premium-features.core :as premium-features])    ;; For Metabase 0.52 or after 
+    'metabase.premium-features.core
+    (catch Exception _
+      (try
+        (require '[metabase.public-settings.premium-features :as premium-features])   ;; For Metabase < 0.52
+        'metabase.public-settings.premium-features
+        (catch Exception e
+          (throw (ex-info "Could not load either premium features namespace"
```

Comment:
what will happen with locally hosted metabase? premium-features exists but is false, so it won't throw??